guide/C/ch_invest.xml

```diff
@@ -1605,7 +1605,7 @@ Income
             <row>
               <entry>Assets:Brokerage Account:Stock:AMZN</entry>
 
-              <entry>-100</entry>
+              <entry>&mdash;100</entry>
```

Comment:
No, use &amp;minus; for negative numbers. em-dashes are the really long ones sometimes used as comma substitutes&mdash;as in this example&mdash;in english sentences.